src/package/store.rs

```diff
@@ -229,6 +234,34 @@ impl PackageStore {
 
         paths
     }
+
+    /// Sync this stores proto files to the vendor directory
+    async fn populate(&self, manifest: &Manifest) -> miette::Result<()> {
```

Comment:
@mara-schulke Not sure about all of this here... Is this the right way of error handling with miette?

Reply:
yes, you should ideally add context to the errors after calling `into_diagnostic` using the `WrapErr` trait